src/patients/notes/NotesList.tsx

```diff
@@ -0,0 +1,50 @@
+import { Alert, List, ListItem } from '@hospitalrun/components'
+import React from 'react'
+import { useHistory } from 'react-router-dom'
+
+import Loading from '../../shared/components/Loading'
+import useTranslator from '../../shared/hooks/useTranslator'
+import Note from '../../shared/model/Note'
+import usePatientNotes from '../hooks/usePatientNotes'
+
+interface Props {
+  patientId: string
+}
+
+const NotesList = (props: Props) => {
+  const { patientId } = props
+  const history = useHistory()
+  const { t } = useTranslator()
+  const { data, status } = usePatientNotes(patientId)
+
+  if (data === undefined || status === 'loading') {
+    return <Loading />
+  }
+
+  if (!data || data.length === 0) {
```

Comment:
I think this can just be simplified to `data.length === 0` since `!data` would have been caught in the check above